pureStorageDriver/templates/plugin/scc.yaml

```diff
@@ -0,0 +1,34 @@
+{{- if eq .Values.plugin.orchestrator openshift}}
```

Comment:
{{- if eq .Values.orchestrator.name "openshift"}}

Reply:
Fair enough, can't hurt to be safe

Reply:
actually it wouldn't work until I changed to this